src/Raven.Server/Documents/Subscriptions/SubscriptionStorage.cs

```diff
@@ -729,5 +750,47 @@ public void RaiseNotificationForBatchEnded(string subscriptionName, Subscription
         {
             OnEndBatch?.Invoke(subscriptionName, batchAggregator);
         }
+
+        public static SubscriptionGeneralDataAndStats GetSubscriptionFromServerStore(ServerStore serverStore, TransactionOperationContext context, string databaseName, string name)
+        {
+            var subscriptionBlittable = serverStore.Cluster.Read(context, SubscriptionState.GenerateSubscriptionItemKeyName(databaseName, name));
```

Comment:
can we introduce serverStore.Cluster.Subscriptions.Read & friends?

Reply:
done